tests/test_ostinato.py

```diff
@@ -137,3 +137,59 @@ def test_deterministic_ostinatoed_with_isconstant(seed, dask_cluster):
         npt.assert_almost_equal(ref_radius, comp_radius)
         npt.assert_almost_equal(ref_Ts_idx, comp_Ts_idx)
         npt.assert_almost_equal(ref_subseq_idx, comp_subseq_idx)
+
+
+@pytest.mark.parametrize(
+    "seed", np.random.choice(np.arange(10000), size=25, replace=False)
```

Comment:
Is there a particular reason why we need to set a random seed?

Reply:
Good point! 
I don't think so. In fact, I now think it might be better to set the seed to a certain value so that we can remove this complexity from the test function. We can better find the issue if it raises an error in the future.

Reply:
I don't think we need to touch/set the random seed at all.

Reply:
@seanlaw 
Is this because of the purpose of this test function? (that we do not need to test the outputs for different inputs but just to see if the input remains unchanged?

Reply:
Yes, that's correct. It's fine to add an `np.nan` in the original input but what we are testing is if the input, `Ts` remains completely unchanged after it is passed through `ostinato`